src/ConfigUpdater.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace Drupal\commerce;
+
+
```

Comment:
Double newlines between the namespace and use statements. Your inner Python developer is trying to break free :P